src/app/onboarding/components/create-new-archive/create-new-archive.component.scss

```diff
@@ -78,53 +80,151 @@ form {
   align-self: center;
 }
 
-.archive-type-description {
+h2 {
+  font-weight: bold;
+  font-size: 1.5em;
+  margin-top: 2.25em;
+  margin-bottom: 10px;
+}
+
+h3 {
+  font-size: 16px;
+  font-weight: bold;
+  line-height: 22px;
+  letter-spacing: 0em;
+  text-align: left;
+}
+
+.select-option {
+  font-family: Open Sans;
+  font-size: 16px;
+  font-style: italic;
+  line-height: 22px;
   color: #646464;
 }
 
-.archive-type {
-  input {
-    visibility: collapse;
+.archive-name {
+  &-container {
+    position: relative;
+    margin-top: 30px;
   }
 
-  img {
-    @media only screen and (max-width: $mobile-breakpoint) {
-      max-height: 5em;
-      margin-bottom: 0;
-      flex: 1;
-    }
-    width: 100%;
-    margin-bottom: 33px;
+  & > h3 {
+    margin-bottom: 0;
   }
 
-  div {
+  &-input {
     display: flex;
-    flex-direction: column;
+    justify-content: center;
+
+    & > span {
+      margin: 10px 20px 0px 20px;
+      font-size: 20px;
+      display: inline-flex;
+      align-items: center;
+    }
+
+    & > input {
+      width: 400px;
+      border: 1px solid #c8c8c8;
+      border-radius: 6px;
+      height: 42px;
+      margin-top: 10px;
+    }
   }
+}
 
-  &:hover {
-    background-color: #FFEAD5;
+.disabled-overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: rgba(255, 255, 255, 0.6);
+  pointer-events: none;
+  z-index: 10;
+}
+
+.buttons {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 50px;
+  & > .btn-link {
+    text-decoration: underline;
+  }
+  & > .btn {
+    width: 250px;
+    margin: 0;
   }
 
-  &.selected {
-    outline: 3px solid #800080;
-    border-color: #800080;
+  & > .btn-disabled {
+    background-color: #c8c8c8;
+    color: #646464;
+    border-color: #c8c8c8;
   }
+}
 
-  @media only screen and (max-width: $mobile-breakpoint) {
-    flex-direction: row;
-    justify-content: center;
+.archive-type {
+  margin-top: 50px;
+}
+
+.explanation {
+  font-family: Open Sans;
+  font-size: 16px;
+  font-style: italic;
+  font-weight: 400;
+  line-height: 22px;
+  letter-spacing: 0em;
+  text-align: left;
+  color: #646464;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 10px; /* This adds a gap between grid items, adjust as necessary */
+
+  @include beforeDesktop {
+    grid-template-columns: repeat(2, 1fr);
   }
+}
+
+.grid-item {
+  border: 2px solid #c8c8c8;
+  border-radius: 10px;
+  padding: 10px;
   display: flex;
-  flex-direction: column;
   align-items: center;
-  border: 1px solid #C8C8C8;
-  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
-  border-radius: 8px;
-  padding: 20px 11px;
-  background-color: #fff;
-  cursor: pointer;
```

Comment:
Can we preserve the `cursor: pointer` rule so these grid squares look clickable on desktop?